.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false
 language: node_js
 
 node_js:
-  - "7.0"
+  - "8.2.1"
```

Comment:
As far as I know VSCode is running on node v7.4.0, so why would run the tests on node 8?